File: KodnestPy/palindrome.py
num = int(input("Enter a number : "))

rev = 0 
temp = num
while temp > 0:
    digit = temp%10
    rev = (rev * 10) + digit
    temp = temp//10
print(rev)
if rev==num:
    print(num," Number is palindrome")
else:
    print(num," Number is not a palindrome")

File: SimplyCoding-master/src/Palindrome.java

public class Palindrome {

	static boolean revNum(int num) {
		int temp = num;
		int sum =0;
		int digit = 0;
		while(num > 0) {
			digit = num%10;
			sum = sum*10+digit;
			num = num/10;
		}
		return temp == sum;
	}
	
//	void revNumber() {
//		for(int i=100;i<150;i++) {
//			int sum=0;
//			int digit = 0;
////			int temp = num;
//			int i = 0;
//		}
//		
//	}

}
